addon/package.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "ember-sinon-qunit",
+  "version": "7.2.0",
+  "description": "Sinon sandbox test integration for QUnit",
+  "keywords": [
+    "ember-addon",
+    "sinon",
+    "qunit"
+  ],
+  "bugs": {
+    "url": "https://github.com/elwayman02/ember-sinon-qunit/issues"
+  },
+  "repository": "https://github.com/elwayman02/ember-sinon-qunit",
+  "license": "MIT",
+  "contributors": [
+    "Jordan Hawker <hawker.jordan@gmail.com> (https://www.JordanHawker.com)",
+    "Steve Calvert <steve.calvert@gmail.com>"
+  ],
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "scripts": {
+    "lint": "concurrently \"npm:lint:*(!fix)\" --names \"lint:\"",
+    "lint:fix": "concurrently \"npm:lint:*:fix\" --names \"fix:\"",
+    "lint:hbs": "ember-template-lint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
+    "prepack": "ember ts:precompile",
+    "postpack": "ember ts:clean"
+  },
+  "dependencies": {
+    "@types/sinon": "^10.0.19",
+    "broccoli-funnel": "^3.0.8",
+    "ember-auto-import": "^2.6.3",
+    "ember-cli-babel": "^8.2.0",
+    "ember-cli-typescript": "^5.2.1"
+  },
+  "devDependencies": {
+    "@tsconfig/ember": "^3.0.1",
+    "concurrently": "^8.0.1",
+    "ember-cli": "~4.11.0",
+    "ember-cli-dependency-checker": "^3.3.1",
+    "ember-cli-htmlbars": "^6.2.0",
+    "ember-cli-inject-live-reload": "^2.1.0",
+    "ember-cli-sri": "^2.1.1",
+    "ember-cli-terser": "^4.0.2",
+    "ember-load-initializers": "^2.1.2",
+    "ember-resolver": "^11.0.1",
+    "ember-try": "^3.0.0",
+    "eslint": "^8.37.0",
+    "eslint-config-prettier": "^9.0.0",
+    "eslint-plugin-ember": "^11.4.9",
+    "eslint-plugin-n": "^16.2.0",
+    "eslint-plugin-prettier": "^5.0.1",
+    "eslint-plugin-qunit": "^8.0.1",
+    "prettier": "^3.0.3",
+    "release-it": "^16.2.1",
+    "typescript": "^5.0.3"
+  },
+  "peerDependencies": {
+    "ember-source": ">=3.28.0",
+    "qunit": "^2.0.0",
+    "sinon": "^15.0.3 || ^16.0.0"
+  },
+  "engines": {
+    "node": "14.* || 16.* || >= 18"
+  },
+  "ember": {
+    "edition": "octane"
+  },
+  "ember-addon": {
+    "after": [
+      "ember-sinon"
```

Comment:
Can you go ahead and drop this? We should have dropped it before, but we no longer rely on ember-sinon so this block is useless.

Reply:
It's also ok for this to be a follow-up, just wanted to call it out while I noticed it.

Reply:
@elwayman02 I removed it